_user_guides/kubernetes-v2.md

```diff
@@ -0,0 +1,113 @@
+
+### Current Limitations
+- S3, SQS, and SNS are not supported for artifact delivery
+- Native Spinnaker deployment strategies such as red/black, highlander, rolling red/black deployment are not supported. If these stragies are desired consider using the deployment object in your manifests.
+- The V2 provider only considers `containers` and `configMaps` for binding to the deployed manifest. `secrets` and other resource types are coming soon.
+- You cannot manually trigger the pipeline, you have to use Github triggers.
+
+### Available Manifest Based Stages
+
+There are 4 stages that are available:
+
+1. **Deploy Manifest** -  Uses `kubectl apply` to deploy a manifest.  Spinnaker will wait until the manifest stabilizes in the Kubernetes cluster or reach the expected capacity on healthy pods.
+
+2. **Delete Manifests** - Removes the manifests and their corresponding artifacts in kubernetes based on different types and labels.  
+
+3. **Scale Manifests** - Scales replica sets to a given static size.
+
+4. **Rollback Manifest** - Allows rollback of a given Kuberentes artifact by a given number of versions.  Helpful in cases of a failed deployment or failed manual QA.
+
+## Setting Up The V2 Provider
+
+Setting up the V2 provider is similar to the [V1 Kubernetes configuration](http://docs.armory.io/admin-guides/configure_kubernetes/#configure-clouddriver-to-use-the-kubectl-config-file) however we'll need to change the provider flag in `/opt/spinnaker/config/clouddriver-local.yml` to `v2`.  For example:
+
+```
+kubernetes:
+  accounts:
+    - name: k8s-v2
+      providerVersion: v2
+      kubeconfigFile: /opt/spinnaker/credentials/kubeconfig
+      dockerRegistries:
+        - accountName: gcr
+```
+
+We'll also need to enable artifact handling in Spinnaker by setting a flag in `/opt/spinnaker/config/spinnaker-local.yml`
+
+```
+features:
+  artifacts:
+    enabled: true
+```
+
+Then restart Armory Spinnaker: `service armory-spinnaker restart`
+
+## Creating a Kubernetes V2 Pipeline
+
+### Configuring The Pipeline Trigger
+We'll begin by creating a pipeline that is triggered from Kuberentes artifacts and delivered through Github.  Below we'll define two artifacts that will be deployed as Kuberenetes manifests: `deployment.yml` and `config-map.yml` which are valid Kubernetes manifests.  Make sure to select the source as `Github`.
+
+![artifacts](https://cl.ly/091z2h232r0d/page.png)
+
+After configuring the artifacts we'll need to associate them with a Github trigger so the pipeline is triggered whenever there are modifications pushed to Github.  For example, the pipeline below will only trigger when _either_ `deployment.yml` _or_ `config-map.yml` are modified and pushed to the repository.
```

Comment:
Maybe call out that if one isn't modified, the latest version of the other is used. Might be obvious, but it's a little confusing at first.

Reply:
Sounds good, i'll add it.